### ISSUE TITLE
Cannot right click taskbar

### ISSUE THREAD

**Issue by CrazyWolf13** [moudey/Shell#433]
I just cannot right click the taskbar anymore, if I right click the taskbar just no menu appears at all.
Microsoft Windows
Version 23H2 (Build 22631.3235)

**Comment by RubicBG** [comment]
do you have 3rd party software to manage or replace taskbar/start menu?

**Comment by CrazyWolf13** [comment]
> do you have 3rd party software to manage or replace taskbar/start menu?

Not really, only translucenttb, but that also happends when it's compledtely off, and I have explorerpatcher

**Comment by RubicBG** [comment]
I've used explorerpatcher before and it doesn't cause a problem. Translucenttb may be the problem and it doesn't have to be active - it may just be that the dll file is being redistributed causing the problem

there is also a way to stop the NS from activating on the taskbar - just need time to check how it was

**Comment by RubicBG** [comment]
> To prevent Shell from replacing the taskbar menu, open Registry and create a subkey with this path
> 
> HKEY_CURRENT_USER\Software\Nilesoft\Shell\Disable
> "Taskbar"=1

**Comment by CrazyWolf13** [comment]
> I've used explorerpatcher before and it doesn't cause a problem. Translucenttb may be the problem and it doesn't have to be active - it may just be that the dll file is being redistributed causing the problem
> 
> there is also a way to stop the NS from activating on the taskbar - just need time to check how it was

Hmm okay I see, yeah I never had problems before, they just appeared.

Compledtely uninstalling translucentTB and rebooting does not solve the issue.

**Comment by CrazyWolf13** [comment]
> > To prevent Shell from replacing the taskbar menu, open Registry and create a subkey with this path
> > HKEY_CURRENT_USER\Software\Nilesoft\Shell\Disable
> > "Taskbar"=1

Thanks, this seems to kinda work, however it has like a 3
5second delay and each time explorer freezes for this time.


Is there any real way to fix this?

**Comment by RubicBG** [comment]
without knowing what provoked it - there is nothing to be done, when mоуdey comes back he may have a better solution

it is no code error related, but check the shell.log - open in text editor and see if there any errors recorded

**Comment by CrazyWolf13** [comment]
> without knowing what provoked it - there is nothing to be done, when mоуdey comes back he may have a better solution
> 
> it is no code error related, but check the shell.log - open in text editor and see if there any errors recorded

Oh that's really sad.

Nope no log entries about that.

If there is anything I can check or report back I'm happy to help :)

**Comment by Athari** [comment]
Taskbar menu (on Windows 10) was unusable for me because it was white on white (or white on transparent...). Setting `theme.dark` to `true` and `theme.background.opacity` to `95` fixed it.
